gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributesTest.java

```diff
@@ -29,14 +31,13 @@
 
   private static final byte[] HAPPY = "(✿◕ ‿◕ )ノ".getBytes(UTF_8);
 
-  @Rule
-  public final AppEngineRule appEngineRule = new AppEngineRule();
 
   private Path path;
   private Path dir;
 
   @Before
   public void before() throws Exception {
+    CloudStorageFileSystemProvider.setGCloudOptions(LocalGcsHelper.options());
```

Comment:
Yes, if we decide to make all test method to either not throw or throw `Exception` (and never
throw the more specific Exception) than my argument is weak but not completely gone.

We do have a tool to find unused throw statement (which in return can also cause for undetected unused import statements). If we decide to disable that check on test classes
how do we make sure that someone didn't actually used a specific Exception?

I would prefer a checkstyle or some other automated tool to tell me that instead of making
it the responsibility of the reviewer but I am unaware of such. Do you?